chapter_natural-language-processing-pretraining/bert-pretraining.md

```diff
@@ -201,13 +201,13 @@ for downstream natural language processing applications.
 ## Summary
 
 * The original BERT has two versions, where the base model has 110 million parameters and the large model has 340 million parameters.
-* After pretraing BERT, we can use it to represent single text, text pairs, or any token in them.
+* After pretraining BERT, we can use it to represent single text, text pairs, or any token in them.
 * In the experiment, the same token has different BERT representation when their contexts are different. This supports that BERT representations are context-sensitive.
 
 ## Exercises
 
 1. In the experiment, we can see that the masked language modeling loss is significantly higher than the next sentence prediction loss. Why?
-1. Set the maximum length of a BERT input sequence to be 512 (same as the original BERT model). Use the configurations of the original BERT model such as $\text{BERT}_{\text{LARGE}}$. Do you encounter any error when running this section? Why?
+2. Set the maximum length of a BERT input sequence to be 512 (same as the original BERT model). Use the configurations of the original BERT model such as $\text{BERT}_{\text{LARGE}}$. Do you encounter any error when running this section? Why?
```

Comment:
why do we change it to 2?

Reply:
Those are 2 different exercises, right? Although both render the same result, but it is better to use numbering.
Anyways the main change was the typo and I changed this to re-trigger the CI.

Reply:
I see. It's a good practice to keep all numbers as 1 so as to make it easy to switch them :)

Reply:
Ohh okay, in that case, I think we should then make this consistent across all chapters. At the moment, I can see a few .md files with all numbers as 1 and a few with 1.2.3...so on.

If all 1s is the convention then let's use it everywhere for consistency.